extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss

```diff
@@ -47,14 +47,21 @@
     border-radius: 100px;
     padding: 0.25rem 0.5rem;
 
-    img {
-      width: 1rem;
-      height: 1rem;
+    .AccountAssets__asset--logo {
+      display: flex;
+      width: 0.875rem;
+      height: 0.875rem;
       border-radius: 1rem;
-      background: var(--pal-brand-primary-on);
```

Comment:
I couldn't quite figure out why this background was needed. Let me know if this needs to come back!

Reply:
Ah right!